docs/farming-&-staking/staking/staking.md

```diff
@@ -127,57 +77,52 @@ Alternatively, you can use [Subscan](https://subspace.subscan.io/) which is a li
+
+![N-nominators-7](/img/doc-imgs/operators-staking/N-Nominators-7.png)
 
 
 Once the withdrawal is submitted, it's finalized after the **domain epoch** is completed. All the withdrawn funds are unlocked after the **locking period** is complete. The current **locking period** is set to 28,800 blocks, or about 48 hours. The locking period is necessary to ensure that the domain block executing the withdrawal is confirmed and not challenged by a fraud proof and to increase the economic stability of domains.
 
-After the **locking period**, the withdrawn amount can be unlocked in the user's account with the `unlock_funds` extrinsic.
+After the **locking period**, the withdrawn amount can be unlocked in the user's account.
+
+8. Using the same operator you used to withdraw your stake from, click on **Actions** -> **Unlock Funds**.
+
+![N-nominators-8](/img/doc-imgs/operators-staking/N-Nominators-8.png)
 
-1. Proceed to [PolkadotJS](https://polkadot.js.org/apps/#/explorer).
-2. Navigate to Developer -> Extrinsics. 
-3. Select the account you want to use in `using the selected account`.
-4. Select `domains` under `submit the following extrinsic` and choose `unlockFinds(operatorId)` in the dropdown.
-5. Provide the `operatorID` you were staking with. 
 6. Submit the transaction, your funds should be unclocked and available once the epoch is complete (up to 10 minutes).
 
-![Staking-29](/img/doc-imgs/operators-staking/Staking-29.png)
-
-### Calculating your nominator balance
-
-1. Proceed to PolkadotJS **[Network Explorer](https://polkadot.js.org/apps/#/explorer)**.
-2. Go to `Developer` -> `Chain state`.
-3. Select `domains` under `selected state query` and `nominators(u64, AccoundId32)`.
-4. Provide the `operatorId` and select your `account` from the dropdown. 
-5. Run the query, remember the `shares` number. 
-  ![Staking-25](/img/doc-imgs/operators-staking/Staking-25.png)
-6. On the same screen, choose `domainStakingSummary(u32)`.
-7. Provide the `domainId`. 
-8. Run the query, remember the `currentTotalStake` number.
-  ![Staking-26](/img/doc-imgs/operators-staking/Staking-26.png)
-9. Without leaving the page, select `operators(u64)`.
-10. Provide `operatorId` that you nominated previously. 
-11. Run the query, remember the `currentTotalStake` number. 
-  ![Staking-27](/img/doc-imgs/operators-staking/Staking-27.png)
-
-To calculate your nominator **balance**:
-1. Calculate **share price** by dividing **currentTotalStake** from the domain by operator **currentTotalStake**.
-2. Multiply **share price** and your nominator **shares** number. 
\ No newline at end of file
+:::note
+You can also withdraw your stake using the Nominators tab.  Your nomination(s) will be the only ones with an action buttin on the last collumn of the grid, ut with so many nominators it can be difficult to find yours.
```

Comment:
```suggestion
You can also withdraw your stake using the Nominators tab. Your nomination(s) will be the only ones with an action button on the last column of the table, but with so many nominators it can be difficult to find yours.
```